.helm/values.yaml

```diff
@@ -39,8 +39,16 @@ service:
 config_content: ""
 
 clickup:
-  webhooksecret: ""
-  token: ""
+  CQYUVTWH2:
+    webhooksecret: ""
+    token: "pk_"
+    list: "10942379"
+    host: "https://api.clickup.com/api/v2"
+  CQTG74FUH:
+    webhooksecret: ""
+    token: "pk_"
+    list: "10942379"
+    host: "https://api.clickup.com/api/v2"
```

Comment:
I think you want to place it to config_content. config_content stored in secret.yaml

Reply:
fixed